### ISSUE TITLE
Questions about the paper

### ISSUE THREAD

**Issue by Sylva-Lin** [VisualComputingInstitute/DITR#1]
Hello,  Thanks for your great work!  
I’ve read your paper, where you distilled a 3D model using Dinov2 and then applied the distilled model to downstream tasks like point cloud semantic segmentation.  
I have a question: When performing point cloud semantic segmentation, do you need to retrain the segmentation head using the dataset labels?

**Comment by kabouzeid** [comment]
Hi, thanks! :) Yes, the semantic segmentation head is trained from scratch during fine-tuning. The distillation task uses a similar head, but with a different number of output channels (to match the DINOv2 feature dimensions) and, of course, a different loss. It is discarded after the distillation pretraining step.
